build-packages/magento-scripts/lib/commands/start.js

```diff
@@ -169,7 +169,11 @@ module.exports = (yargs) => {
                 process.exit(0);
             } catch (e) {
                 logger.error(e.message || e);
-                await googleAnalytics.trackError(e.message || e);
+                if (e && e.reportAnalytics) {
```

Comment:
This check will skip all non-CMA install errors. Please redo this check to skip if report analytics is set to false.

I.e. type errors will not be caught.

Reply:
This is currently a draft PR, I was planning to make some custom error classes to prevent reporting of errors like "your docker is installed, restart your system" which is the only way of gracefully stopping task flow. So, something like type error will still be caught, as well as other non-cma errors.
So, this will be developed a little bit later, but my idea is like that.